README.md

```diff
@@ -1,3 +1,7 @@
+# Notice
+We strongly recommend using the [databend-driver](https://github.com/databendlabs/bendsql/blob/main/bindings/python/README.md) as it provides more comprehensive features.
```

Comment:
Maybe change this link to `https://pypi.org/project/databend-driver`?

Reply:
this is better, pypi is the homepage for python eggs